### DISCUSSION TITLE
Data中文件格式

### DISCUSSION THREAD

**Discussion by MeAPikachu** [physics-data/faq#64] [Q&A] [answered]
<img width="261" alt="Screenshot 2021-08-23 at 20 24 54" src="https://user-images.githubusercontent.com/61261422/130446623-d37b782b-d0fd-478a-a323-123dd9480742.png">

将文件后缀强行更改为txt之后可以打开查看答案（为了之后的评测的顺利运行，需要修改回ans）
为什么使用.ans 而不是.txt 文件呢？

**Comment by Harry-Chen** [comment, marked as answer]
用于表明这是答案文件

**Reply by MeAPikachu** [reply]
3ans.txt ？

**Reply by Harry-Chen** [reply]
并没有硬性规定 “文本文件的后缀必须是 txt“ 甚至 ”文本文件必须有后缀“。建议你使用编辑器（无论是 VS Code 还是 vim）打开项目文件夹，它们不会拒绝任何后缀的文件，就不会遇到这类问题了。

**Reply by MeAPikachu** [reply]
啊，是的)！
双击打开方法不是最好的
